src/osgPlugins/OpenFlight/Pools.cpp

```diff
@@ -125,7 +125,7 @@ osg::Material* MaterialPool::getOrCreateMaterial(int index, const osg::Vec4& fac
     osg::Material* poolMaterial = get(index);
 
     // Create final material.
-    osg::Material* material = osg::clone(poolMaterial, osg::CopyOp::SHALLOW_COPY);
+    osg::Material* material = dynamic_cast<osg::Material*>(poolMaterial->clone(osg::CopyOp::SHALLOW_COPY));
```

Comment:
I think we can use the 3.6 version of this file.

Reply:
Reverted changes to this file (pending)